pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -236,6 +238,23 @@ func (opt *Options) Run(args []string) error {
 	// anomalies
 	var syntheticTestResults []*JUnitTestCase
 	if events := m.Events(time.Time{}, time.Time{}); len(events) > 0 {
+		// Serialize the interval data for easier external analysis
+		intervalsFilename := filepath.Join(opt.JUnitDir, "intervals.json.gz")
+		if file, err := os.Create(intervalsFilename); err != nil {
+			fmt.Fprintf(opt.Out, "ERROR: failed to create %s: %v\n", intervalsFilename, err)
+		} else {
+			gz := gzip.NewWriter(file)
```

Comment:
Don't gzip automatically, you should just output the raw file.

Reply:
I think it's like 3mb vs. 300kb. I removed the compression